Bermuda/Bermuda/InteractableNPC.cpp

```diff
@@ -0,0 +1,78 @@
+#include "InteractableNPC.h"
+
+InteractableNPC::InteractableNPC(int id, int healthPoints, int attackPoints, int actionRange, int attackRange, Spawnpoint *spawnPoint, int interactStartX, int interactStartY, int interactWitdh, int interactHeight, long attackTime) :
+	NPC( id, actionRange, attackRange, spawnPoint ),
+	Entity( id, spawnPoint->getX(), spawnPoint->getY() ),
+	InteractableEntity(id, spawnPoint->getX(), spawnPoint->getY(), interactStartX, interactStartY, interactWitdh, interactHeight ),
+	AttackingNPC(attackTime)
```

Comment:
ik heb hier inderdaad niet zo goed over nagedacht en ook met het switchen van states is dit erg nou verbonden